akka-sample-cqrs-scala/src/main/scala/sample/cqrs/Main.scala

```diff
@@ -5,12 +5,20 @@ import java.util.concurrent.CountDownLatch
 
 import scala.concurrent.Await
 import scala.concurrent.duration._
-
 import akka.actor.typed.ActorSystem
 import akka.actor.typed.Behavior
 import akka.actor.typed.scaladsl.Behaviors
+import akka.cluster.sharding.typed.{
+  ClusterShardingSettings,
+  ShardedDaemonProcessSettings
+}
```

Comment:
weird format

Reply:
I agree, default scalafmt I think, I have fmt on save enabled so that's probably why.

Reply:
I have intellij configured to by default use `<path of akka/akka>/.scalafmt`

Reply:
One Akka format to rule them all!